charts/rstudio-pm/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if and .Values.config.Metrics.Enabled .Values.serviceMonitor.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "rstudio-pm.fullname" . }}
+{{- if .Values.serviceMonitor.namespace }}
+  namespace: {{ .Values.serviceMonitor.namespace }}
```

Comment:
I personally am more fond of the pattern using `default` rather than `if/else` (worth noting that the ordering of `default` is always hard to get right 🙈 ):
```
namespace: {{ .Values.serviceMonitor.namespace | default $.Release.Namespace }}
```